client/homebrew/pages/basePages/listPage/listPage.jsx

```diff
@@ -0,0 +1,189 @@
+require('./listPage.less');
```

Comment:
Double-check for any unused functions here. `userPage.jsx` had a lot of unused stuff left over (I went through and removed them). I expect `listPage.jsx` similarly has a couple unused functions left behind after the refactor.

Reply:
The only things I'm not quite clear on is the `.noColumns()` - it looks like it's just a way to avoid having to re-enter the same hunk of CSS; however it's only used in one place, so it's not actually avoiding any repetition.
At this point, I haven't removed it.